src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private StringBuilder builder;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate localDateFrom = stringToDate(dateFrom);
+        LocalDate localDateTo = stringToDate(dateTo);
+        LocalDate localDateFromRecord;
+        builder = new StringBuilder();
+        int salary;
+        builder.append("Report for period ");
+        builder.append(dateFrom).append(" - ");
+        builder.append(dateTo);
+        for (String name : names) {
+            salary = 0;
+            builder.append("\n").append(name).append(" - ");
```

Comment:
[Use System.lineSeparator](https://github.com/mate-academy/jv-salary-info/blob/master/checklist.md#use-systemlineseparator)
```suggestion
            builder.append("\n").append(name).append(" - ");
```